### PR TITLE
Add better list types to function map

### PR DESCRIPTION
**Pull request by rvanvelzen**

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
Thank you very much, I really appreciate your work a lot :)

File: resources/functionMap.php [hunk 1]
```diff
@@ -11692,14 +11692,14 @@
 'stomp_version' => ['string'],
 'StompException::getDetails' => ['string'],
 'StompFrame::__construct' => ['void', 'command='=>'string', 'headers='=>'array', 'body='=>'string'],
-'str_getcsv' => ['array', 'input'=>'string', 'delimiter='=>'string', 'enclosure='=>'string', 'escape='=>'string'],
+'str_getcsv' => ['non-empty-list<?string>', 'input'=>'string', 'delimiter='=>'string', 'enclosure='=>'string', 'escape='=>'string'],
```
Comment (ondrejmirtes) [inline resources/functionMap.php:11695]: Why is the value nullable? I've got a related bug report around this: https://github.com/phpstan/phpstan/issues/9393

Comment (rvanvelzen) [reply resources/functionMap.php:11695]: https://3v4l.org/EJhab

Comment (ondrejmirtes) [reply resources/functionMap.php:11695]: Thank you!

**Pull request merged**